### PR TITLE
Fix loading models from MlflowCatalog

### PR DESCRIPTION
**Pull request by eddyxu**
With this PR, Rikai supports to directly run `SELECT ML_PREDICT(model, image)` over an existing model from mlflow.

### CODE REVIEW

File: python/rikai/spark/sql/codegen/mlflow_registry.py [hunk 1]
```diff
@@ -224,7 +224,8 @@ def get_model_conf(self, spec, run):
 
     def get_options(self, spec, run):
         options = run.data.params
-        options.update(spec.get("options", {}))
+        if spec.get("options", {}):
```
Comment (changhiskhan) [inline python/rikai/spark/sql/codegen/mlflow_registry.py:227]: nit: why the change here?

**Review by changhiskhan — APPROVED** [review]

File: python/rikai/spark/sql/codegen/mlflow_registry.py (hunk 1 above)
Comment (eddyxu) [reply python/rikai/spark/sql/codegen/mlflow_registry.py:227]: `spec` now is passed by jvm to python for the jdbc change, which is possible to be serialized as `{"options": None}` now. 

It was originally handled by the jvm code when passing through `py4j` to change `None` to `{}`.
